llm/gguf.go

```diff
@@ -67,9 +65,21 @@ const (
 
 type kv map[string]any
 
+type tensor struct {
+	name   string
+	kind   uint32
+	offset uint64
+	size   uint64
+	ne     [4]uint64
```

Comment:
what is ne here?

Reply:
ne is the number of elements in each dimension

Reply:
maybe `numElems`? It was definitely ambiguous.

Reply:
I changed it to `shape` which should be more descriptive and added a comment